scenarios/mono.benchmarks.yml

```diff
@@ -0,0 +1,234 @@
+# Examples:
+# --config mono.benchmarks.yml --scenario plaintext-mono-jit --profile aspnet-citrine-lin
+
+imports:
+  - https://raw.githubusercontent.com/aspnet/Benchmarks/master/src/WrkClient/wrk.yml
+
+jobs:
+  platformbenchmarks:
+    source:
+      repository: https://github.com/aspnet/benchmarks.git
+      branchOrCommit: master
+      project: src/BenchmarksApps/Kestrel/PlatformBenchmarks/PlatformBenchmarks.csproj
+    readyStateText: Application started.
+    framework: net5.0
+
+  postgresql:
+    source:
+      repository: https://github.com/TechEmpower/FrameworkBenchmarks.git
+      branchOrCommit: master
+      dockerFile: toolset/databases/postgres/postgres.dockerfile
+      dockerImageName: postgres_te
+      dockerContextDirectory: toolset/databases/postgres
+    readyStateText: ready to accept connections
+    noClean: true
+
+scenarios:
+  plaintext-mono-jit:
+    application:
+      job: platformbenchmarks
+      useMonoRuntime: jit
+    load:
+      job: wrk
+      variables:
+        presetHeaders: plaintext
+        path: /p
+        serverPort: 5000
+        pipeline: 16
+
+  plaintext-mono-llvm-jit:
+    application:
+      job: platformbenchmarks
+      useMonoRuntime: llvm-jit
+    load:
+      job: wrk
+      variables:
+        presetHeaders: plaintext
+        path: /p
+        serverPort: 5000
+        pipeline: 16
+
+  plaintext-mono-llvm-aot:
+    application:
+      job: platformbenchmarks
+      useMonoRuntime: llvm-aot
+    load:
+      job: wrk
+      variables:
+        presetHeaders: plaintext
+        path: /p
+        serverPort: 5000
+        pipeline: 16
+
+  json-mono-jit:
+    application:
+      job: platformbenchmarks
+      useMonoRuntime: jit
+    load:
+      job: wrk
+      variables:
+        presetHeaders: json
+        path: /j
+        serverPort: 5000
+        connections: 512
+
+  fortunes-mono-jit:
```

Comment:
From the presentation in the .NET Town Hall, it is a goal to improve on Json, Fortunes and other benchmarks for CoreCLR. I believe Mono should do the same, and we should add new scenarios and reporting for all the one CoreCLR focusses heavily on.